src/store/getters/state-getters.ts

```diff
@@ -50,6 +50,11 @@ export const getEffectiveDate = (state: any): Date => {
   return state.stateModel.incorporationDateTime.effectiveDate
 }
 
+/** The filing's original effective date. */
+export const getOriginalEffectiveDate = (state: any): Date => {
```

Comment:
This returns a string, not a Date object.

Reply:
fixed 👍